edweiss-firebase/functions/src/functions/fcm/registerFCMToken.ts

```diff
@@ -0,0 +1,26 @@
+import { FCMCommunication } from 'model/users';
+import { onAuthentifiedCall } from 'utils/firebase';
+import { Collections, getDocumentAndRef } from 'utils/firestore';
+import { fail, ok } from 'utils/status';
+
+export const registerFCMToken = onAuthentifiedCall(FCMCommunication.Functions.registerFCMToken, async (userId, args) => {
+
+
+    if (!userId)
+        return fail("invalid_userID");
+    if (!args.fcmToken)
+        return fail("invalid_fcm_tokens");
+
+    const [user, userRef] = await getDocumentAndRef(Collections.users, userId);
```

Comment:
You can add a try/catch, this can help you catch and respond to potential issues during database interactions.

Reply:
Hmmm I am not so sure if it is actually very useful in this scenario but I will check again thank you